2024-08-09 06:08:06,165 - distributed.nanny - INFO -         Start Nanny at: 'tcp://10.34.59.4:42122'
2024-08-09 06:08:06,169 - distributed.nanny - INFO -         Start Nanny at: 'tcp://10.34.59.4:37141'
2024-08-09 06:08:06,763 - distributed.diskutils - INFO - Found stale lock file and directory '/tmp/dask-scratch-space/worker-7954igau', purging
2024-08-09 06:08:06,763 - distributed.diskutils - INFO - Found stale lock file and directory '/tmp/dask-scratch-space/worker-s5lz0jug', purging
2024-08-09 06:08:07,283 - distributed.worker - INFO -       Start worker at:     tcp://10.34.59.4:38491
2024-08-09 06:08:07,283 - distributed.worker - INFO -       Start worker at:     tcp://10.34.59.4:43803
2024-08-09 06:08:07,283 - distributed.worker - INFO -          Listening to:     tcp://10.34.59.4:38491
2024-08-09 06:08:07,283 - distributed.worker - INFO -           Worker name:           SLURMCluster-1-0
2024-08-09 06:08:07,283 - distributed.worker - INFO -          Listening to:     tcp://10.34.59.4:43803
2024-08-09 06:08:07,283 - distributed.worker - INFO -           Worker name:           SLURMCluster-1-1
2024-08-09 06:08:07,283 - distributed.worker - INFO -          dashboard at:           10.34.59.4:42237
2024-08-09 06:08:07,283 - distributed.worker - INFO - Waiting to connect to:     tcp://10.34.59.1:39609
2024-08-09 06:08:07,283 - distributed.worker - INFO -          dashboard at:           10.34.59.4:43151
2024-08-09 06:08:07,283 - distributed.worker - INFO - -------------------------------------------------
2024-08-09 06:08:07,283 - distributed.worker - INFO - Waiting to connect to:     tcp://10.34.59.1:39609
2024-08-09 06:08:07,283 - distributed.worker - INFO -               Threads:                          1
2024-08-09 06:08:07,283 - distributed.worker - INFO - -------------------------------------------------
2024-08-09 06:08:07,283 - distributed.worker - INFO -                Memory:                   1.86 GiB
2024-08-09 06:08:07,283 - distributed.worker - INFO -               Threads:                          1
2024-08-09 06:08:07,283 - distributed.worker - INFO -       Local Directory: /tmp/dask-scratch-space/worker-tdinbm9y
2024-08-09 06:08:07,283 - distributed.worker - INFO -                Memory:                   1.86 GiB
2024-08-09 06:08:07,283 - distributed.worker - INFO -       Local Directory: /tmp/dask-scratch-space/worker-pdu907ym
2024-08-09 06:08:07,283 - distributed.worker - INFO - -------------------------------------------------
2024-08-09 06:08:07,283 - distributed.worker - INFO - -------------------------------------------------
2024-08-09 06:08:07,695 - distributed.worker - INFO - Starting Worker plugin shuffle
2024-08-09 06:08:07,696 - distributed.worker - INFO -         Registered to:     tcp://10.34.59.1:39609
2024-08-09 06:08:07,696 - distributed.worker - INFO - -------------------------------------------------
2024-08-09 06:08:07,696 - distributed.core - INFO - Starting established connection to tcp://10.34.59.1:39609
2024-08-09 06:08:07,696 - distributed.worker - INFO - Starting Worker plugin shuffle
2024-08-09 06:08:07,697 - distributed.worker - INFO -         Registered to:     tcp://10.34.59.1:39609
2024-08-09 06:08:07,697 - distributed.worker - INFO - -------------------------------------------------
2024-08-09 06:08:07,697 - distributed.core - INFO - Starting established connection to tcp://10.34.59.1:39609
slurmstepd: error: *** JOB 2330403 ON cl-node004 CANCELLED AT 2024-08-09T06:08:43 ***
2024-08-09 06:08:43,033 - distributed.worker - INFO - Stopping worker at tcp://10.34.59.4:38491. Reason: scheduler-close
2024-08-09 06:08:43,033 - distributed.worker - INFO - Stopping worker at tcp://10.34.59.4:43803. Reason: scheduler-close
2024-08-09 06:08:43,034 - distributed.batched - INFO - Batched Comm Closed <TCP (closed) Worker->Scheduler local=tcp://10.34.59.4:56326 remote=tcp://10.34.59.1:39609>
Traceback (most recent call last):
  File "/users/stud/k/kruret00/.local/lib/python3.10/site-packages/distributed/comm/tcp.py", line 297, in write
    raise StreamClosedError()
tornado.iostream.StreamClosedError: Stream is closed

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/users/stud/k/kruret00/.local/lib/python3.10/site-packages/distributed/batched.py", line 115, in _background_send
    nbytes = yield coro
  File "/users/stud/k/kruret00/.local/lib/python3.10/site-packages/tornado/gen.py", line 767, in run
    value = future.result()
  File "/users/stud/k/kruret00/.local/lib/python3.10/site-packages/distributed/comm/tcp.py", line 307, in write
    convert_stream_closed_error(self, e)
  File "/users/stud/k/kruret00/.local/lib/python3.10/site-packages/distributed/comm/tcp.py", line 142, in convert_stream_closed_error
    raise CommClosedError(f"in {obj}: {exc}") from exc
distributed.comm.core.CommClosedError: in <TCP (closed) Worker->Scheduler local=tcp://10.34.59.4:56326 remote=tcp://10.34.59.1:39609>: Stream is closed
